buffer/src/main/java/io/netty5/buffer/ByteBufUtil.java

```diff
@@ -130,6 +132,40 @@ public static Buffer writeAscii(BufferAllocator alloc, CharSequence seq) {
         return alloc.copyOf(seq.toString().getBytes(StandardCharsets.US_ASCII));
     }
 
+    /**
+     * Create a copy of the underlying storage from {@code buf} into a byte array.
+     * The copy will start at {@link Buffer#readerOffset()} and copy {@link Buffer#readableBytes()} bytes.
+     */
+    public static byte[] getBytes(Buffer buf) {
+        return getBytes(buf,  buf.readerOffset(), buf.readableBytes());
+    }
+
+    /**
+     * Create a copy of the underlying storage from {@code buf} into a byte array.
+     * The copy will start at {@code start} and copy {@code length} bytes.
+     */
+    public static byte[] getBytes(Buffer buf, int start, int length) {
+        return getBytes(buf, start, length, true);
+    }
+
+    /**
+     * Return an array of the underlying storage from {@code buf} into a byte array.
+     * The copy will start at {@code start} and copy {@code length} bytes.
+     * If {@code copy} is true a copy will be made of the memory.
+     * If {@code copy} is false the underlying storage will be shared, if possible.
```

Comment:
@chrisvest seems like we not do this yet ?

Reply:
Good catch. This is a copy-paste error.